test/server/api/cache/url.test.ts

```diff
@@ -0,0 +1,59 @@
+import 'reflect-metadata'
```

Comment:
sorry to revive this PR, but how come every test file need to import this?

Reply:
consider using jest's setupfile array that sets up everything nicely to reduce boilerplate https://jestjs.io/docs/en/configuration#setupfiles-array

Reply:
good catch. can touch this up in a subsequent PR.

Reply:
Addressed in #84